Lib/getpass.py

```diff
@@ -40,7 +42,7 @@ def unix_getpass(prompt='Password: ', stream=None):
 
     Always restores terminal settings before returning.
     """
-    passwd = None
+    passwd = ""
```

Comment:
Please revert. A password can be empty.

Reply:
yea, but in this case we return an empty string

Reply:
we also using `passwd = ""` in `win_getpass`

Reply:
> yea, but in this case we return an empty string

And that's a valid password. Even if it's a weak one, it's still a valid password.